### PR TITLE
📝(project) improve main doc page and developer guide

### PR DESCRIPTION
**Pull request by wilbrdt**
## Purpose

Following @lebaudantoine review of our documentation, improving main documentation page and developer guide.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -23,13 +23,41 @@
...
-- **command-line interface** (CLI), to build data pipelines the UNIX-way™️,
-- **library**, to fetch learning events from various backends, (de)serialize or
+- **[LRS](./tutorials/lrs.md)**, an HTTP API server to collect xAPI statements (learning events), following the [ADL LRS standard](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Communication.md#partthree)
+- **[command-line interface](./tutorials/cli.md)** (CLI), to build data pipelines the UNIX-way™️,
+- **[library](./tutorials/library.md)**, to fetch learning events from various backends, (de)serialize or
     convert them from and to various standard formats such as
     [xAPI](https://adlnet.gov/projects/xapi/), or
     [openedx](https://docs.openedx.org/en/latest/developers/references/internal_data_formats/tracking_logs/index.html)
 
+## What is an LRS?
```
Comment (quitterie-lcs) [inline docs/index.md:33]: You can add an info admonition at the end of this subsection to inform readers that the new documentation is the following one: https://opensource.ieee.org/xapi. However, Ralph still supports last 1.X version of xAPI

Comment (wilbrdt) [reply docs/index.md:33]: Great idea, thanks!

**Review by quitterie-lcs — APPROVED** [review]

File: docs/index.md [hunk 2]
```diff
@@ -23,13 +23,47 @@
 
 Ralph is a toolbox for your learning analytics, it can be used as a:
 
-- **[LRS](https://en.wikipedia.org/wiki/Learning_Record_Store)**, a HTTP API server to collect xAPI statements (learning events), following the [ADL LRS standard](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Communication.md#partthree)
-- **command-line interface** (CLI), to build data pipelines the UNIX-way™️,
-- **library**, to fetch learning events from various backends, (de)serialize or
+- **[LRS](./tutorials/lrs.md)**, an HTTP API server to collect xAPI statements (learning events), following the [ADL LRS standard](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Communication.md#partthree)
```
Comment (SergioSim) [inline docs/index.md:26]: broken link? (./tutorials/lrs.md)

Comment (quitterie-lcs) [inline docs/index.md:26]:
```suggestion
- **[LRS](./tutorials/lrs.md)**, an HTTP API server to collect xAPI statements (learning events), following the [ADL LRS standard](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Communication.md#part-three-data-processing-validation-and-security)
```

Comment (wilbrdt) [reply docs/index.md:26]: Thanks!

Comment (wilbrdt) [reply docs/index.md:26]: For now yes... I can remove it until lrs tutorial is added though!

**Review by SergioSim — APPROVED** [review]
Looks great)
image

File: docs/index.md [hunk 3]
```diff
@@ -23,13 +23,47 @@
...
     [openedx](https://docs.openedx.org/en/latest/developers/references/internal_data_formats/tracking_logs/index.html)
 
+## What is an LRS?
+
+A Learning Record Store, or LRS, is a key component in the context of learning analytics and the Experience API (xAPI).
+
+The [Experience API](https://github.com/adlnet/xAPI-Spec) (or Tin Can API) is a standard for tracking and reporting learning experiences. 
+In particular, it defines:
+
+- the [xAPI format](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Data.md#part-two-experience-api-data) of the learning events. xAPI statements includes an `actor` (e.g. *learner*), a `verb` (e.g. *watched*, *passed*) and an `object` (e.g. *video*, *quiz*) as well as contextual and optional information (e.g. *timestamp* of the event, *result* of an activity). Here's a simplistic statement:
```
[the remark this answers is not in the document]
Comment (wilbrdt) [reply docs/index.md:40]: Removed the examples to shorten the line!

**Pull request merged**